bridetalk_project/src/components/pages/masterlayout.css

```diff
@@ -0,0 +1,23 @@
+
+
+.img {
+background-image: url('../../../public/images/image\ 2.jpg');
+ /* background-repeat: no-repeat; */
+  
+  /* Set rules to fill background */
+  min-height: 100%;
+  min-width: 1024px;
+
+  /* Proportionate scaling */
+  width: 100%;
+  height: auto;
+
+  /* Positioning */
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.routes{
+  font-family: Inter-bold;
```

Comment:
We had a talk about fonts the previous time.
Now that you managed to import the fonts from Google, make sure u delete these occurences.